Student Performance Analyzer code

AbhiramHN opened the pull request:
Here is the code for Student Performance Analyzer, if any corrections are there please let me know!

AbhiramHN committed:
Pushed the student performance analyzer program

DK-25-TECH reviewed (changes requested):
mate as of now use appropriate camel case and pascal case name where ever required i could see many mismatch names and try to make single return statement in the functions

AbhiramHN committed:
Updated the function and variable names, and fixed issue with mmultiple return statement

AbhiramHN committed:
Removed .exe file from tracking

AbhiramHN committed:
Added const keyword and added the switch case

Ramyaurs02 commented on StudentPerformanceAnalyzer.c:
```diff
@@ -0,0 +1,120 @@
+#include <stdio.h>
+
+struct Student
```
use typedef struct

Ramyaurs02 commented on StudentPerformanceAnalyzer.c:
```diff
@@ -0,0 +1,120 @@
+    int mark1, mark2, mark3;
+};
+
+char GradeCalculator(const float averageMarks)
```
use camelcase for names

Ramyaurs02 commented on StudentPerformanceAnalyzer.c:
```diff
@@ -0,0 +1,120 @@
+
+const char *GetStar(const char grade)
+{
+    const char *stars;
```
instead of making character star, try to return number of stars to print use that in a for loop to print the stars

Ramyaurs02 commented on StudentPerformanceAnalyzer.c:
```diff
@@ -0,0 +1,120 @@
+        scanf("%d %s %d %d %d", &students[index].rollNumber, students[index].name,
+              &students[index].mark1, &students[index].mark2, &students[index].mark3);
+    }
+
```
this should be in a separate function

Ramyaurs02 commented on StudentPerformanceAnalyzer.c:
```diff
@@ -0,0 +1,120 @@
+        }
+    }
+
+    for (int index = 0; index < numberOfStudents; index++)
```
this should be in a separate method

Ramyaurs02 commented on StudentPerformanceAnalyzer.c:
```diff
@@ -0,0 +1,120 @@
+              &students[index].mark1, &students[index].mark2, &students[index].mark3);
+    }
+
+    for (int i = 0; i < numberOfStudents - 1; i++)
```
this should be in a separate function

AbhiramHN committed:
Created seperate functions for codes in main, and added typedef and used camelcase

AbhiramHN committed:
Delete StudentPerformanceAnalyzer.exe

AbhiramHN commented:
I have changed the code according to your review. I changed all names to camelcase. Added typedef to structure. And created separate function for each task that i was doing in main(). If any changes are there please let me know. **I made 2 commits because i missed braces in 1st commit.**

AbhiramHN committed:
Added braces to ifelse and switch which i missed earlier nothing changed except for that

AbhiramHN committed:
Merge branch 'StudentPerformance' of https://github.com/AbhiramHN/Kalpavriksha-Assignments into StudentPerformance
